README.md

```diff
@@ -168,6 +169,14 @@ fullcontact.person.email('foo@bar.com', null, 'https://mycallbackurl.com', 'webh
 });
 ```
 
+The `Person` API's now accept the `macromeasures` param as a boolean
+
+```js
+fullcontact.person.email('opensource@observe.it', null, null, null, null, null, true, function (err, data) {
```

Comment:
Not related to this PR, but maybe we should start using an object instead of `null, null, null, null, true` for API. This is getting really verbose and unreadable. Thoughs @AdriVanHoudt ?